From 603e66fa176460be358cabfa1d78040267541320 Mon Sep 17 00:00:00 2001
From: Naman-1234 <namankalrabhiwani54@gmail.com>
Date: Tue, 19 May 2020 18:27:47 +0000
Subject: [PATCH] Create Patch File

---
 client3.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/client3.py b/client3.py
index a2558ad..cf879c2 100644
--- a/client3.py
+++ b/client3.py
@@ -2,6 +2,8 @@ import urllib.request
 import time
 import json
 import random
+import unittest
+from client3 import *
 QUERY = "http://localhost:8080/query?id={}"
 N=500
 def getDataPoint(quote):
-- 
2.17.1

